packages/cli/src/server/server.js

```diff
@@ -23,6 +23,9 @@ function buildCommand(yargs) {
       type: 'number',
       default: 9001,
     },
+    listenHost: {
```

Comment:
I can't think of what else we would use `host` for on the server, let's just use `host`?

Reply:
I first thought of using `host` but I thought it would bring some ambiguity with the `url` param. But I will change it to `host`.

Reply:
> I thought it would bring some ambiguity with the url param.

Oh like from the client-side `serverBaseUrl`? good thinking :) I'm not as worried about that because they're separate commands with their own namespaces, so I think we should be ok with `host` 👍